### ISSUE TITLE
Visit concept vs type concept

### ISSUE THREAD

**Issue by MaximMoinat** [ohdsi-studies/ConceptPrevalence#3]
For visit occurrences, the `visit_type_concept_id` is counted and not the `visit_concept_id`? It might not be the most interesting coding (just a few standard concept ids), but I think it is good to include the `visit_concept_id` as well for completeness.

https://github.com/ohdsi-studies/ConceptPrevalence/blob/b6461ccd8ee3506196ea7b3edd85c3585a5f962e/inst/sql/sql_server/ConceptPrevalence.sql#L78-L85

**Comment by aostropolets** [comment]
Thanks a lot for the suggestion! How do you think we can then analyze such data?

**Comment by MaximMoinat** [comment]
Probably the same as we do for the the other `<domain>_concept_id`s. That being said, the visit domain will not be the most interesting, as there are just a few standard concepts. Most will be inpatient, outpatient or emergency room visit. 

The most interesting analysis would be to see if people use non-standard or even custom concepts to define their visits. For some data sources the standard visit concepts do not always capture the visits.

**Comment by aostropolets** [comment]
Agree, custom concepts would be super-interesting but we made a deliberate decision not to include custom mappings in the analysis as they technically proprietary and do not belong to the open source network.
I'll add visit_concept_id to the analysis, but we may need a specific research use case to justify custom mappings collection.
